docs/user/developers/index.rst

```diff
@@ -40,10 +40,12 @@ carry out common development tasks in Dash.
    :maxdepth: 1
 
    integration.rst
-   translating.rst
+   integration-apis
+   integration-sdks
    compiling.rst
    testnet.rst
    insight.rst
+   translating.rst
 
 .. _understanding-sporks:
```

Comment:
Let's move sporks out to a separate page as well? It seems out of place here, strangely specific for a top level overview page otherwise containing only general information

Reply:
Makes sense. I've moved it out to it's own page.